src/token.spec.js

```diff
@@ -48,15 +48,15 @@ describe('token', () => {
       await xo2.call('token.delete', {
         token
       })
-      xo2.close()
 
-      await getConnection({credentials: {token}}).then(
-        (res) => {
-          res.close()
+      await testConnection({credentials: {token}}).then(
+        async () => {
+          await xo2.close()
           throw new Error('xo2.signIn should have thrown')
         },
-        function (error) {
+        async function (error) {
           expect(error.code).toEqual(3)
+          await xo2.close()
```

Comment:
You have duplicate code here, take a look at https://github.com/JsCommunity/golike-defer